setSceneASCII.h

```diff
@@ -0,0 +1,4 @@
+#include "dialogueMaker/dialogueMaker.h"
+#include <iostream>
+
+void SetASCII();
```

Comment:
I don't see where SetASCII get called? Where it will be used?

Reply:
it's called inside  the Main function .

Reply:
okie